### DISCUSSION TITLE
License?

### DISCUSSION THREAD

**Discussion by victorkane** [theodorusclarence/ts-nextjs-tailwind-starter#158] [Q&A] [answered]
Starter looks awesome, is just what I need! But, how do you recommend actually using this great starter for an actual project, if there is no license? Thanks in advance!

**Comment by theodorusclarence** [comment, marked as answer]
Hi @victorkane, great question. This project is MIT Licensed, so feel free to use it on your actual project!

I didn't put the LICENSE file on the repo so users don't need to remove the default license on every project init.
